Detectors/EMCAL/calibration/testWorkflow/EMCALChannelCalibratorSpec.h

```diff
@@ -442,8 +442,8 @@ DataProcessorSpec getEMCALChannelCalibDeviceSpec(const std::string calibType, co
 
   // data request needed for rejection of EMCal trigger
   if (rejectL0Trigger) {
-    inputs.emplace_back(device::getCTPConfigBinding(), "CTP", "CTPCONFIG", 0, Lifetime::Condition, ccdbParamSpec("CTP/Config/Config", ctpcfgperrun));
-    inputs.emplace_back(device::getCTPDigitsBinding(), "CTP", "DIGITS", 0, Lifetime::Timeframe);
+    inputs.emplace_back(device::getCTPConfigBinding(), "CTP", "CTPCONFIG", inputsubspecCTP, Lifetime::Condition, ccdbParamSpec("CTP/Config/Config", ctpcfgperrun));
```

Comment:
Confgurable subspec does not impact CCDB objects - should stay with 0.

Reply:
Here we have however a different problem: we can have multiple CTP configs per run, as AliECS allows multiple runs to be in parallel. We needs somehow a protection that the object for the proper run is selected. We might be able to do this in the finalizeCCDB.